src/test/java/com/google/cloud/tools/appengine/operations/cloudsdk/serialization/GcloudStructuredLogTest.java

```diff
@@ -99,8 +98,10 @@ public void testParse_inputMalformed() {
     try {
       GcloudStructuredLog.parse("non-JSON");
       fail();
-    } catch (JsonParseException e) {
-      assertThat(e.getMessage(), CoreMatchers.containsString("JsonSyntaxException"));
+    } catch (JsonParseException expected) {
+      String message = expected.getMessage();
```

Comment:
Seems like this should remain assertThat (with new package) with containsString, would remove need for null check.

Reply:
done